lib/rubycritic/core/rating.rb

```diff
@@ -1,6 +1,16 @@
+require "colorize"
+
 module Rubycritic
 
   class Rating
+    RATING_TO_COLOR = {
```

Comment:
This colouring logic should be on `ConsoleReport`, not here - a private method like `color_for_rating` or something like that and pass both `mod` and the color to the template :)